must_triage/inspectors/ocp.py

```diff
@@ -78,3 +78,24 @@ def pod_ready(obj):
                 f"Container '{cs['name']}' in "
                 f"pod '{obj['metadata']['name']}' is not ready")
         return result
+    @staticmethod
+    def restart_count(obj):
+        result = list()
+        status = obj['status']
+        try:
+            container_statuses = status['containerStatuses']
+        except KeyError:
+            return [dict(
+                pod_name=obj['metadata']['name'],
+                status=status,
+                )]
+        not_ready = filter(lambda cs: cs['restartCount'] != '0', container_statuses)
```

Comment:
Might be more appropriate to call the resulting object `restarted` as opposed to `not_ready`

Reply:
> Thanks for the PR! I have some suggestions, but I also think it's worth clarifying what's desired here: is it "any container that has restarted", "any container that has tried to restart and _failed_", or something else?
> 
> To test this, of course it'd be ideal to have a must-gather bundle that exhibits the bad behavior you want to detect; if that's not available, perhaps just copying an existing must-gather bundle and modifying one or more of the yaml files to "pretend" to exhibit the bad behavior.
To clarify you, we are expecting the restart count of any container which have restarted.